src/Driver/Pdo/PdoDriverInterface.php

```diff
@@ -78,7 +78,7 @@ public function getUsername(): string;
      *
      * @param string $password The password for establishing DB connection.
      */
-    public function password(string $password): void;
+    public function password(#[\SensitiveParameter] string $password): void;
```

Comment:
Attribute not need in interface. Attribute should be added in implementation. Example: https://3v4l.org/KNALl

Reply:
So you'd prefer to see this in AbstractPdoDriver class that implements PdoDriverInterface?

Reply:
Yes

Reply:
I believe this is resolved, thanks for the help!

Reply:
Add to constructor parameter also. And need line in changelog.